.github/workflows/publish-oss-feature-branch.yml

```diff
@@ -364,11 +378,54 @@ jobs:
         "worker",
         "k8s_runner",
       ]
+    outputs:
+      found: ${{ steps.nscheck.outputs.found }}
     steps:
       - uses: actions/checkout@v3
-      - name: Install helm chart
+      - name: Find the namespace
+        id: nscheck
         uses: WyriHaximus/github-action-helm3@v3
         with:
```

Comment:
technically helm upgrades can be done with `helm upgrade --install` but other steps and jobs needs to know explicitly.